cmd-car-split.go

```diff
@@ -65,6 +66,15 @@ type carFile struct {
 	fileSize   int64
 }
 
+type Metadata struct {
```

Comment:
can we add byte size values so that given a general offset, we can calculate which subset file it corresponds to?

Should `fileSize` maybe be enough?

what do you think @anjor ?

Reply:
Yeah `fileSize` should be enough for this. What else would the byte size value be?

Given that this metadata is for retrievals when **not** using filecoin, i don't think we need to worry about any padding.